pipeline-scripts/buildlib.groovy

```diff
@@ -3,7 +3,21 @@ def commonlib = load("pipeline-scripts/commonlib.groovy")
 
 commonlib.initialize()
 
-def initialize() {
+def path_setup() {
```

Comment:
A more robust pattern in Jenkins pipelines is
```
withEnv(["PATH+SOMETHING=${env.WORKSPACE'}/somepath"]) {
      sh 'sometool'
}
```

I'd prefer to see this explicitly in each pipeline instead of changing the path globally in the library.

Reply:
Ok!

I'll leave the entry to `build-scripts/ose_images` that was already there though - only moving PATH / GOPATH setup into a function to keep init a bit cleaner.